BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/provider/EddystoneLocationProvider.java

```diff
@@ -1,23 +1,21 @@
 package com.nexenio.bleindoorpositioning.location.provider;
 
-import com.nexenio.bleindoorpositioning.ble.advertising.AdvertisingPacket;
 import com.nexenio.bleindoorpositioning.ble.beacon.Eddystone;
-import com.nexenio.bleindoorpositioning.location.Location;
 
 /**
  * Created by steppschuh on 16.11.17.
  */
 
-public class EddystoneLocationProvider extends BeaconLocationProvider<Eddystone> {
+public class EddystoneLocationProvider<B extends Eddystone> extends BeaconLocationProvider<B> {
 
-    public EddystoneLocationProvider(Eddystone beacon) {
+    public EddystoneLocationProvider(B beacon) {
         super(beacon);
     }
 
     @Override
     public void updateLocation() {
-        AdvertisingPacket advertisingPacket = beacon.getLatestAdvertisingPacket();
-        location = new Location();
+        //AdvertisingPacket advertisingPacket = beacon.getLatestAdvertisingPacket();
```

Comment:
oh, we should either remove or use it

Reply:
Depends on the implementation / setup of beacons. There's currently no specification for this.

Reply:
What do you mean? Currently we don't need this obviously. So why keeping this comment?

Reply:
If someone ever wants to implement the logic, that's the code they'll need anyway. No need to keep it, though.

Reply:
Ok than I would suggest to remove it. It should be easy to find the code via git, right? Btw. no need to do it in this PR, we could do it on dev